ROADMAP.md

```diff
@@ -1,56 +1,47 @@
-# KF Serving 2019 Roadmap
-## Q3 2019
-### v0.2 Integrate with the ML Ecosystem (ETA: August 15, 2019)
-Objective: "Continue to simplify the user experience by deeply integrating with the Kubeflow Ecosystem."
-* Kubeflow Integration
-    * Prepare KFServing to release v0.2 and v0.3 alongside Kubeflow v0.7.
-    * Integrate with `kfctl generate` and `kfctl apply`.
-    * Deploy as a [Kubernetes Application](https://github.com/kubernetes-sigs/application).
-    * Integrate with Kubeflow Pipelines to enable model deployment from a Pipeline.
-    * Integrate with Fairing to enable model deployment from a Notebook.
-    * Achieve 20% End-to-End Test Coverage of Supported Features. (See v0.3 for 80%).
-    * Support PVCs to enable integration with on-prem Kubeflow installations.
-    * Document Installation for various cloud providers (GCP, IBM Cloud, Azure, AWS).
+# KF Serving Roadmap
+## 2019 - 2020
 
-Objective: "Empower users to deeply understand their predictions and validate KFServing's static graph architecture."
-* Explainability
-    * Deploy a predictor and explainer, powered by Alibi.
-    * Deploy a predictor and explainer, powered by user specified explainer container.
+### v0.4 Performance (ETA: Jan 31, 2019)
+Objective: "Prevent performance regressions across a known set of representative models."
+* Automated Performance Tests 
+    * Define a set of Models to test covering a wide array of usecases and frameworks.
+    * Publish performance results over time to enable regression tracking.
 
-Objective: "Increase coverage of ML frameworks to support previously unsupported customer workloads."
-* Frameworks
-    * Deploy a ONNX model
-    * Explore supporting other model serialization mechanisms for certain frameworks (e.g. saving PyTorch models with dill)
+Objective: "Enable users to deploy latency sensitive models with KFServing."
+* High Performance Dataplane
+    * Enable support for GRPC or similar.
+    * Continue to support existing HTTP Dataplane.
+
+Objective: "Reduce Total Cost of Ownership when deploying multiple underutilized models."
+* GPU Sharing 
+    * Reduce TCO by enabling models of the same framework and version to be co-hosted in a single model server.
+
+### v0.3 Stability (ETA: Dec 15, 2019)
+Objective: "Improve practices around dependency management." 
+* Migrate to Kubebuilder 2.0.
+    * Use Go Modules.
+    * Stop Vendoring dependencies.
+    * Avoid the extremely heavy dependency on Tensorflow.
+* Migrate to Kubernetes 1.15.
+    * Enable LabelSelectors for the Pod Mutation Webhook.
```

Comment:
Does it mean that kfserving will not support k8s version before 1.15?

Reply:
we are going to upgrade to use 1.15 apis but I do not see any reason why it can not work with previous kubernetes though.

Reply:
If we take a hard dependency on K8s 1.15, we may rely on APIs that don't exist in 1.14

Reply:
this might be a bummer - lot of enterprise k8s still not on 1.15 ...